database/migrations/2020_12_12_203153_migrate_admin_user.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+use App\Models\Configs;
+use App\Models\User;
+use Illuminate\Database\Migrations\Migration;
+
+class MigrateAdminUser extends Migration
+{
+	/**
+	 * Run the migrations.
+	 *
+	 * @return void
+	 */
+	public function up()
+	{
```

Comment:
Why doesn't this also remove `username` and `password` from Config?

Reply:
Failsafe. Just in case something goes wrong, it is easier to keep the data than to recreate them. We could delete them at some point.